### ISSUE TITLE
Publisher cannot approve and publish workflow submissions

### ISSUE THREAD

**Issue by charliestrafe** [engram-design/Workflow#153]
### Describe the bug

I have an issue where my publisher users (set as workflow publisher user groups) cannot publish any submissions as it results in a 400 error as seen in the following screenshot.

![Screenshot 2022-07-13 at 08 55 57](https://user-images.githubusercontent.com/47243887/178683318-a6583609-3036-4efb-9379-b13c8e7bc994.png)

### Steps to reproduce

1.  Submit an item for review from an editor user
2.  Attempt to 'approve and publisher' as a publisher group.

### Craft CMS version

Craft Pro 4.1.2

### Plugin version

dev-craft-4 as 2.0.0-beta.4

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/workflow` requirement in `composer.json` to:

```
"require": {
  "verbb/workflow": "dev-craft-4 as 2.0.0-beta.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by charliestrafe** [comment]
This is now working

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.5](https://github.com/verbb/workflow/releases/tag/2.0.0-beta.5)
